apps/minaintyg/src/pages/Certificate/CertificatePage.tsx

```diff
@@ -0,0 +1,43 @@
+import { IDSSpinner } from '@frontend/ids-react-ts'
+import { skipToken } from '@reduxjs/toolkit/query'
+import { useParams } from 'react-router-dom'
+import { DisplayHTML } from '../../components/DisplayHTML/DisplayHTML'
+import { PageDivider } from '../../components/PageDivider/PageDivider'
+import { PageHeading } from '../../components/PageHeading/PageHeading'
+import { useGetCertificateQuery } from '../../store/api'
+import { CertificateBody } from './components/CertificateBody/CertificateBody'
+import { CertificateFooter } from './components/CertificateFooter'
+import { CertificateInformation } from './components/CertificateInformation'
+import { CertificateStatusBadge } from './components/CertificateStatusBadge'
+
+export function CertificatePage() {
+  const { id } = useParams()
+  const { data: certificate, isLoading } = useGetCertificateQuery(id ? { id } : skipToken)
+
+  if (isLoading) {
+    return <IDSSpinner data-testid="spinner" />
+  }
+
+  if (!certificate) {
+    return null
+  }
+
+  return (
+    <>
+      <PageHeading heading={certificate.metadata.type.name}>
+        <DisplayHTML html={certificate.metadata.type.description} />
```

Comment:
Since description will come in a later stage, maybe we can just return a "TODO" value, or just ignore this for now.